c/zis/server.c

```diff
@@ -1166,7 +1166,7 @@ static int loadConfig(ZISContext *context,
             ZIS_LOG_INPUT_PARM_MSG, mainParms);
     zowedump(NULL, LOG_COMP_STCBASE, ZOWE_LOG_INFO,
              (void *)mainParms,
```

Comment:
As I read this, you are including the binary halfword length of the parms in the data that is being written, which does not seem correct to me.  I think what you want is-
zowedump(NULL, LOG_COMP_STCBASE, ZOWE_LOG_INFO,
                (void *)(mainParms+sizeof(mainParms->textLength),
                 mainParms->textLength);

Reply:
@gadyck the initial intention was to dump the length halfword too.

Reply:
Okay.  It seemed odd, but I did not know exactly what zowedump did.